@hemantmm the test is failing, could you fix it?

```
io.kestra.plugin.core.http.DownloadTest > contentDispositionWithSpaceAfterDot() failed ❌ in 0.082</summary>

```
java.lang.AssertionError: &#10;Expecting actual:&#10;  "kestra:///io/kestra/plugin/core/http/downloadtest/contentdispositionwithspaceafterdot/executions/1JsaBaDikWmEzANfR0Xn4G/tasks/downloadtest/69afdwIqnIiqneJtjMuQ7z/file.with+spaces.txt"&#10;to end with:&#10;  "file.with spaces.txt"&#10;

java.lang.AssertionError: 
Expecting actual:
  "kestra:///io/kestra/plugin/core/http/downloadtest/contentdispositionwithspaceafterdot/executions/1JsaBaDikWmEzANfR0Xn4G/tasks/downloadtest/69afdwIqnIiqneJtjMuQ7z/file.with+spaces.txt"
to end with:
  "file.with spaces.txt"

	at io.kestra.plugin.core.http.DownloadTest.contentDispositionWithSpaceAfterDot(DownloadTest.java:236)
```

--- a/core/src/test/java/io/kestra/plugin/core/http/DownloadTest.java
+++ b/core/src/test/java/io/kestra/plugin/core/http/DownloadTest.java
@@ -233,7 +233,7 @@ class DownloadTest {
         Download.Output output = task.run(runContext);
 
         assertThat(output.getUri().toString()).doesNotContain("/secure-path/");
-        assertThat(output.getUri().toString()).endsWith("file.with spaces.txt");
+        assertThat(output.getUri().toString()).endsWith("file.with+spaces.txt");
     }
 
     @Controller()

